src/store/courses/thunk.js

```diff
@@ -0,0 +1,32 @@
+import {
+	addCourse,
+	deleteCourse,
+	getCourses,
+	updateCourse,
+} from '../../services';
+import {
+	store_getCourses,
+	store_addCourse,
+	store_deleteCourse,
+	store_updateCourse,
+} from './actionCreators';
+
+export const thunk_getCourses = () => async (dispatch) => {
+	await getCourses().then((response) => {
+		dispatch(store_getCourses(response.data.result));
+	});
+};
+
+export const thunk_deleteCourse = (id, token) => async (dispatch) => {
+	await deleteCourse(id, token).then(() => dispatch(store_deleteCourse(id)));
```

Comment:
you should dispatch getCourse info, this will update the store and reflected on UI

Reply:
Understood